network-api/networkapi/wagtailpages/pagemodels/blog/blog.py

```diff
@@ -122,8 +129,10 @@ class BlogPage(FoundationMetadataPageMixin, Page):
         FieldPanel('tags'),
     ]
 
-    settings_panels = Page.settings_panels + [
+    settings_panels = [
+        PublishingPanel(),
         FieldPanel('first_published_at'),
+        PrivacyModalPanel(),
```

Comment:
This feels like it would impact everything, not just blogs?

Reply:
Nope, just the BlogPage because it's only being set on the `BlogPage` class.